com.unity.render-pipelines.high-definition/Editor/VFXGraph/VFXHDRPBinder.cs

```diff
@@ -1,6 +1,7 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using NUnit.Framework;
```

Comment:
Not sure it needed (I guess you added it during testing)

Reply:
Resolved at 1dae0162bd9bc1ae5b4baef1827c8f28e3858865